core/runtime/src/main/java/io/quarkus/runtime/configuration/ApplicationPropertiesConfigSource.java

```diff
@@ -25,7 +25,9 @@
     private static final long serialVersionUID = -4694780118527396798L;
 
     static final String APPLICATION_PROPERTIES = "application.properties";
+    static final String APPLICATION_PROPERTIES_PROFILE = "application.%s.properties";
     static final String MP_PROPERTIES = "META-INF/microprofile-config.properties";
+    static final String MP_PROPERTIES_PROFILE = "META-INF/microprofile-config-%s.properties";
```

Comment:
I find the difference between `application.xxx.properties` vs `microprofile-config-xxx.properties` quite weird. IMHO, it should be either `-` in both cases, or `.` in both cases. (I'd personally prefer `-`, but don't really mind.)

Reply:
Well, you are right. As I added the `microprofile-config-xxx.properties` later following Smallrye implementation, I didn't realize it would be better for both to follow the same pattern. I'm going to fix this.